backend/src/status.ts

```diff
@@ -0,0 +1,34 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from 'aws-lambda';
+import { Lambda } from "aws-sdk"
+
+const lambda = new Lambda();
+
+export const handler = async (_event: APIGatewayProxyEvent, _context: Context): Promise<APIGatewayProxyResult> => {
+  try {
+    const response = await lambda.invoke({
+      FunctionName: 'Essentially-Summary-BE-statusfunction',
+      InvocationType: 'RequestResponse',
+    }).promise()
+
+    const payload = JSON.parse(response.Payload.toString())
+
+    const status = payload.statusCode
+
+    if (status === 200) {
+        return {
+            statusCode: 200,
+            body: "Everything is essentially running :)"
+        }
+    } else {
+        return {
+            statusCode: status || 500,
+            body: "Essentially is active but Summary API is down :("
+        }
+    }
+  } catch {
+    return {
+      statusCode: 500,
+      body: "Essentially is active but Summary API is down :("
```

Comment:
Correct me if I am wrong, but it looks like the message here should be "Essentially is down..."?

Reply:
agreed. It was like this for the express api but i think it should be that

Reply:
pushed new commit with change